src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -463,93 +423,170 @@ protected void InitializeReferences()
-                    runner.runtextpos = runtextpos;
+                RunInternal(input, startat, ref state, callback, runner, usingStringOverload: true, reuseMatchObject);
+            }
+            finally
+            {
+                runner.runtext = null; // drop reference to text to avoid keeping it alive in a cache.
+                _runner = runner;
+            }
+        }
+
+        /// <summary>
+        /// Performs the matching against a given span input, which will call the passed in <paramref name="callback"/> for every
+        /// match that is found, along with the relevant <paramref name="state"/>.
+        /// </summary>
+        internal void Run<TState>(ReadOnlySpan<char> input, int startat, ref TState state, MatchCallback<TState> callback, bool reuseMatchObject)
+        {
+            Debug.Assert((uint)startat <= (uint)input.Length);
+
+            RegexRunner runner = Interlocked.Exchange(ref _runner, null) ?? CreateRunner();
+            try
+            {
+                RunInternal(input, startat, ref state, callback, runner, usingStringOverload: false, reuseMatchObject);
+            }
+            finally
+            {
+                _runner = runner;
+            }
+        }
+
+        /// <summary>
+        /// Performs the internal loop which will search for matches on the given <paramref name="input"/>, and will call the <paramref name="callback"/> for
+        /// each match. If <paramref name="usingStringOverload"/> is set to <see langword="true"/>, then we may need to do additional cleanup which ensures that
+        /// we are not keeping a reference to the input text alive in a cache.
+        /// </summary>
+        private void RunInternal<TState>(ReadOnlySpan<char> input, int startat, ref TState state, MatchCallback<TState> callback, RegexRunner runner, bool usingStringOverload, bool reuseMatchObject)
+        {
+            runner.InitializeTimeout(internalMatchTimeout);
+            int runtextpos = startat;
+            while (true)
+            {
+                runner.InitializeForScan(this, input, startat, false);
+                runner.runtextpos = runtextpos;
 
-                    int stoppos = RightToLeft ? 0 : input.Length;
+                int stoppos = RightToLeft ? 0 : input.Length;
 
-                    Match? match = InternalPerformScan(reuseMatchObject, input, 0, runner, input, returnNullIfQuick: false);
-                    Debug.Assert(match is not null);
+                // We get the Match by calling Scan. 'input' parameter is used to set the Match text which is only relevante if we are using the Run<TState> string
+                // overload, as APIs that call the span overload (like Count) don't require match.Text to be set, so we pass null in that case.
+                Match? match = RunInternal(reuseMatchObject, input: usingStringOverload ? runner.runtext : null, 0, runner, input, returnNullIfQuick: false);
```

Comment:
I'm getting lost in the call graph here.  If I call Count(string), what does the call stack look like by the time I end up in Scan?

Reply:
Also, I'm getting confused by the various internal/private members, some of which are Run and some of which are RunInternal.  What's the distinction for when we use which naming?

Reply:
I understand the confusion and agree that something should be done here to fix that. Here is a verbose explanation of why I have things the way I do as it stands:

As you know, before my previous Span PR we originally had only two Run methods in Regex:

```c#
public class Regex
{
  // This method will try to find a match on the given input and return it back.
  internal Match? Run(string input, ...)
  {
  }

  // This method will try to find all matches on the given input (or until callback returns false), and call
  // the callback for every match found.
  internal void Run<TState>(string input, MatchCallback<TState> callback, ....)
  {
  }
}
```

It is worth noting that these two Run methods had a lot of code duplication internally. On my previous Span PR, I introduced a new Run overload that was almost the same as the first one, except that it would take ReadOnlySpan<char> input as a parameter instead of the string, and this overload wouldn't be setting runtext as all the operations through that overload would be happening over the passed in span. As part of those changes, I also refactored the code that would remove all of the code duplication between all three of the Run methods, and extracted it into a new RunInternal method.

```diff
public class Regex
{
  // internal members
  internal Match? Run(string input, ...);
+ internal Match? Run(ReadOnlySpan<char> input, ...); //<-- Takes span as input and doesn't set runtext.
  internal void Run<TState>(string input, MatchCallback<TState> callback, ....);

  // private members only used to avoid code duplication
+ private static Match? RunInternal(); // <-- Used as refactoring to remove all code duplication from the above three Run methods
}
```

With this PR, I'm adding a new Run<TState> overload that takes in ReadOnlySpan<char> as input and which is not setting runtext (just like the Run overload added by the previous PR), and I'm also abstracting all of the shared code between both of the Run<TState> methods into a new private RunInternal<TState> method.

```diff
public class Regex
{
  // internal members
  internal Match? Run(string input, ...);
  internal Match? Run(ReadOnlySpan<char> input, ...); //<-- Takes span as input and doesn't set runtext.
  internal void Run<TState>(string input, MatchCallback<TState> callback, ....);
+ internal void Run<TState>(ReadOnlySpan<char> input, MatchCallback<TState> callback, ....); //<-- Takes span as input and doesn't set runtext.

  // private members only used to avoid code duplication
  private static Match? RunInternal(); // <-- Used as refactoring to remove all code duplication from the above three Run methods
+ private void RunInternal<TState>(); // <-- Used as refactoring to remove all code duplication from the above three Run<TState> methods
}
```

> I'm getting lost in the call graph here. If I call Count(string), what does the call stack look like by the time I end up in Scan?

```
Regex.Count(string)
-> Regex.Run<TState>(string, ...)
  -> Regex.RunInternal<TState>(ReadOnlySpan<char>, ...) // <-- This is being added by this PR and is just a way of sharing the code between the different  Run<TState> methods
    -> RegexRunner.Scan(ReadOnlySpan<char>)
```

In order to fix the confusion, I wouldn't necessarily suggest a change of design, but for sure at least a rename of methods. For instance, both of the `Run<TState>` methods should probably be renamed to something like `RunForAllMatches<TState>` in order to further differentiate between what they do vs what regular `Run` methods do. Also, we should rename the `RunInternal` methods which are used only to avoid code duplication to something that better suits it and that doesn't create confusion with the actual Run methods.

Alternatively if we wanted to reduce the amount of overloads we have, we could also have only one overload of `Run` and one for `Run<TState>` that works for both the string case and the span case, which would then have to take as parameters a nullable string and a span, and would perform a quick runtime check to see if we are being called by a span API or a string API, for instance:

```c#
  internal Match? Run(bool quick, int prevlen, string? inputString, ReadOnlySpan<char> inputSpan, int beginning, int length, int startat)
    {
        // .. parameter validation

        bool isUsingStringAPIs = inputString is not null;

        RegexRunner runner = Interlocked.Exchange(ref _runner, null) ?? CreateRunner();

        if (isUsingStringAPIs)
        {
            runner.runtext = inputString;
        }

        // .. the rest of the implementation and use isUsingStringAPIs blocks for things that should only happen when using strings
    }
```

Reply:
Thanks.

Let's start by:
- Choosing better names.  Your RunAllMatches suggestion, or RunAllMatchesWithCallback, or something like that, sounds good to me.
- Running some perf tests to make sure that the extra levels being introduced aren't resulting in measurable cost.  Try some short patterns/inputs that will quickly match.

If there's no measurable impact, we can go ahead with it.

> It is worth noting that these two Run methods had a lot of code duplication internally

While duplication in general is undesirable, in certain cases it can be warranted.  Just something to keep in mind.  There's also opportunity to avoid duplication via a helper used in just a few places and marked as AggressiveInlining.

Reply:
I ran the `Perf_regex_industry_RustLang_Sherlock` perf tests and the results show that we are pretty much the same before and after my change. Here are the results:

``` ini

BenchmarkDotNet=v0.13.1.1702-nightly, OS=Windows 11 (10.0.22000.527/21H2)
Intel Core i7-8700 CPU 3.20GHz (Coffee Lake), 1 CPU, 12 logical and 6 physical cores
.NET SDK=7.0.100-preview.2.22110.1
  [Host]     : .NET 7.0.0 (7.0.22.10302), X64 RyuJIT
  Job-IXUFSP : .NET 7.0.0 (42.42.42.42424), X64 RyuJIT
  Job-OTMQBB : .NET 7.0.0 (42.42.42.42424), X64 RyuJIT

PowerPlanMode=00000000-0000-0000-0000-000000000000  Arguments=/p:DebugType=portable,-bl:benchmarkdotnet.binlog  IterationTime=250.0000 ms  
MaxIterationCount=20  MinIterationCount=15  WarmupCount=1  

```
| Method |        Job |         Toolchain |                              Pattern |         Options |            Mean |           Error |          StdDev |          Median |             Min |             Max | Ratio | RatioSD |      Gen 0 |  Allocated | Alloc Ratio |
|------- |----------- |------------------ |------------------------------------- |---------------- |----------------:|----------------:|----------------:|----------------:|----------------:|----------------:|------:|--------:|-----------:|-----------:|------------:|
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                           **(?i)Holmes** |            **None** |    **666,360.8 ns** |     **7,123.43 ns** |     **6,314.73 ns** |    **663,793.3 ns** |    **658,696.5 ns** |    **681,012.0 ns** |  **1.00** |    **0.00** |    **13.3690** |    **97138 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                           (?i)Holmes |            None |    672,447.4 ns |     6,129.39 ns |     5,733.44 ns |    672,211.4 ns |    661,198.7 ns |    684,035.3 ns |  1.01 |    0.01 |    13.2626 |    97138 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                           **(?i)Holmes** |        **Compiled** |    **429,718.7 ns** |     **5,712.03 ns** |     **5,343.04 ns** |    **427,316.0 ns** |    **423,398.1 ns** |    **439,914.7 ns** |  **1.00** |    **0.00** |    **15.2027** |    **97137 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                           (?i)Holmes |        Compiled |    426,228.1 ns |     4,081.17 ns |     3,817.53 ns |    425,830.6 ns |    417,790.1 ns |    431,228.1 ns |  0.99 |    0.01 |    13.8889 |    97137 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                           **(?i)Holmes** | **NonBacktracking** |    **679,686.2 ns** |     **7,340.09 ns** |     **6,506.80 ns** |    **681,543.7 ns** |    **667,303.8 ns** |    **689,825.9 ns** |  **1.00** |    **0.00** |    **13.6240** |    **97138 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                           (?i)Holmes | NonBacktracking |    679,059.5 ns |     6,280.57 ns |     5,874.85 ns |    679,817.4 ns |    670,010.4 ns |    689,494.3 ns |  1.00 |    0.01 |    13.6240 |    97138 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |             **(?i)Sher[a-z]+\|Hol[a-z]+** |            **None** |  **5,789,825.6 ns** |   **114,175.66 ns** |   **112,135.76 ns** |  **5,768,112.5 ns** |  **5,648,765.9 ns** |  **6,006,215.9 ns** |  **1.00** |    **0.00** |    **22.7273** |   **144992 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |             (?i)Sher[a-z]+\|Hol[a-z]+ |            None |  5,879,259.9 ns |    48,678.30 ns |    43,152.04 ns |  5,877,665.3 ns |  5,810,594.4 ns |  5,954,594.4 ns |  1.02 |    0.02 |          - |   144996 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |             **(?i)Sher[a-z]+\|Hol[a-z]+** |        **Compiled** |    **816,355.0 ns** |     **5,803.35 ns** |     **5,428.46 ns** |    **816,047.4 ns** |    **807,930.7 ns** |    **827,038.4 ns** |  **1.00** |    **0.00** |    **20.8333** |   **144979 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |             (?i)Sher[a-z]+\|Hol[a-z]+ |        Compiled |    822,791.6 ns |    13,710.89 ns |    11,449.22 ns |    820,326.2 ns |    806,085.6 ns |    850,127.6 ns |  1.01 |    0.02 |    20.8333 |   144979 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |             **(?i)Sher[a-z]+\|Hol[a-z]+** | **NonBacktracking** |  **1,258,008.3 ns** |    **10,890.19 ns** |     **9,093.80 ns** |  **1,256,039.1 ns** |  **1,240,483.2 ns** |  **1,271,388.3 ns** |  **1.00** |    **0.00** |    **20.3046** |   **144983 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |             (?i)Sher[a-z]+\|Hol[a-z]+ | NonBacktracking |  1,249,940.4 ns |    11,269.25 ns |     9,989.89 ns |  1,248,530.3 ns |  1,238,169.7 ns |  1,270,782.6 ns |  0.99 |    0.01 |    20.5128 |   144980 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **(?i)Sherlock** |            **None** |    **149,652.6 ns** |       **964.00 ns** |       **854.56 ns** |    **149,597.6 ns** |    **148,325.8 ns** |    **151,036.0 ns** |  **1.00** |    **0.00** |     **2.9904** |    **21216 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         (?i)Sherlock |            None |    141,626.3 ns |     1,634.78 ns |     1,529.17 ns |    141,523.6 ns |    140,058.3 ns |    145,372.1 ns |  0.95 |    0.01 |     3.3576 |    21216 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **(?i)Sherlock** |        **Compiled** |    **107,434.2 ns** |       **996.64 ns** |       **832.24 ns** |    **107,186.1 ns** |    **106,389.8 ns** |    **108,926.0 ns** |  **1.00** |    **0.00** |     **2.9762** |    **21216 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         (?i)Sherlock |        Compiled |    101,985.5 ns |     1,221.65 ns |     1,082.96 ns |    101,735.5 ns |    100,091.9 ns |    103,993.7 ns |  0.95 |    0.01 |     3.2895 |    21216 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **(?i)Sherlock** | **NonBacktracking** |    **153,374.6 ns** |     **1,328.46 ns** |     **1,242.64 ns** |    **153,682.0 ns** |    **150,873.5 ns** |    **155,070.6 ns** |  **1.00** |    **0.00** |     **3.0979** |    **21216 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         (?i)Sherlock | NonBacktracking |    146,927.0 ns |       887.86 ns |       830.51 ns |    146,655.4 ns |    145,731.0 ns |    148,640.9 ns |  0.96 |    0.01 |     2.9533 |    21216 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                  **(?i)Sherlock Holmes** |            **None** |    **151,301.0 ns** |     **2,458.23 ns** |     **1,919.22 ns** |    **151,466.4 ns** |    **147,316.6 ns** |    **154,170.8 ns** |  **1.00** |    **0.00** |     **3.0175** |    **19968 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                  (?i)Sherlock Holmes |            None |    143,986.7 ns |     1,146.20 ns |     1,016.08 ns |    143,814.9 ns |    142,074.6 ns |    146,047.0 ns |  0.95 |    0.02 |     2.8952 |    19968 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                  **(?i)Sherlock Holmes** |        **Compiled** |    **104,984.0 ns** |     **1,027.74 ns** |       **911.07 ns** |    **105,251.6 ns** |    **102,769.6 ns** |    **106,157.6 ns** |  **1.00** |    **0.00** |     **2.9362** |    **19968 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                  (?i)Sherlock Holmes |        Compiled |    101,184.2 ns |     1,279.54 ns |     1,196.88 ns |    100,796.7 ns |     99,546.8 ns |    103,570.1 ns |  0.96 |    0.01 |     2.8409 |    19968 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                  **(?i)Sherlock Holmes** | **NonBacktracking** |    **161,283.6 ns** |     **2,790.38 ns** |     **2,473.60 ns** |    **160,401.8 ns** |    **158,570.1 ns** |    **165,612.2 ns** |  **1.00** |    **0.00** |     **3.1827** |    **19968 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                  (?i)Sherlock Holmes | NonBacktracking |    156,065.8 ns |     1,419.69 ns |     1,327.97 ns |    155,921.4 ns |    154,390.7 ns |    158,178.6 ns |  0.97 |    0.02 |     3.1056 |    19968 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |           **(?i)Sherlock\|Holmes\|Watson** |            **None** |  **9,794,695.2 ns** |   **189,541.85 ns** |   **186,155.44 ns** |  **9,714,255.8 ns** |  **9,539,557.7 ns** | **10,108,119.2 ns** |  **1.00** |    **0.00** |          **-** |   **135228 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |           (?i)Sherlock\|Holmes\|Watson |            None |  9,695,464.8 ns |   189,509.67 ns |   158,249.15 ns |  9,665,292.3 ns |  9,499,719.2 ns | 10,032,476.9 ns |  0.99 |    0.02 |          - |   135230 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |           **(?i)Sherlock\|Holmes\|Watson** |        **Compiled** |  **1,788,267.5 ns** |    **19,892.16 ns** |    **18,607.14 ns** |  **1,789,524.3 ns** |  **1,757,704.9 ns** |  **1,825,781.9 ns** |  **1.00** |    **0.00** |    **20.8333** |   **135205 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |           (?i)Sherlock\|Holmes\|Watson |        Compiled |  1,784,462.8 ns |    18,030.97 ns |    16,866.18 ns |  1,789,307.6 ns |  1,751,363.9 ns |  1,808,779.2 ns |  1.00 |    0.01 |    20.8333 |   135205 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |           **(?i)Sherlock\|Holmes\|Watson** | **NonBacktracking** |  **3,665,316.6 ns** |    **23,441.02 ns** |    **21,926.75 ns** |  **3,664,555.1 ns** |  **3,617,453.6 ns** |  **3,697,227.5 ns** |  **1.00** |    **0.00** |    **14.4928** |   **135210 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |           (?i)Sherlock\|Holmes\|Watson | NonBacktracking |  3,671,719.7 ns |    29,819.50 ns |    27,893.18 ns |  3,671,171.6 ns |  3,603,962.7 ns |  3,721,752.2 ns |  1.00 |    0.01 |    14.9254 |   135219 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **(?i)Sherlock\|(...)er\|John\|Baker [49]** |            **None** | **32,730,944.0 ns** |   **414,287.69 ns** |   **323,448.58 ns** | **32,845,057.1 ns** | **32,092,885.7 ns** | **33,123,071.4 ns** |  **1.00** |    **0.00** |          **-** |   **156775 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | (?i)Sherlock\|(...)er\|John\|Baker [49] |            None | 32,973,990.5 ns |   605,790.69 ns |   537,017.67 ns | 32,914,441.7 ns | 32,198,250.0 ns | 33,967,533.3 ns |  1.01 |    0.02 |          - |   156800 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **(?i)Sherlock\|(...)er\|John\|Baker [49]** |        **Compiled** |  **2,722,969.0 ns** |    **49,629.57 ns** |    **46,423.53 ns** |  **2,728,759.4 ns** |  **2,667,550.0 ns** |  **2,830,434.4 ns** |  **1.00** |    **0.00** |    **20.8333** |   **156637 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | (?i)Sherlock\|(...)er\|John\|Baker [49] |        Compiled |  2,739,383.8 ns |    54,090.75 ns |    45,168.23 ns |  2,729,825.5 ns |  2,687,577.6 ns |  2,840,601.6 ns |  1.01 |    0.01 |    20.8333 |   156636 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **(?i)Sherlock\|(...)er\|John\|Baker [49]** | **NonBacktracking** |  **5,029,853.6 ns** |    **78,356.23 ns** |    **69,460.76 ns** |  **5,034,877.6 ns** |  **4,912,493.9 ns** |  **5,129,112.2 ns** |  **1.00** |    **0.00** |    **20.4082** |   **156646 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | (?i)Sherlock\|(...)er\|John\|Baker [49] | NonBacktracking |  5,030,880.3 ns |    45,571.32 ns |    42,627.44 ns |  5,031,923.4 ns |  4,955,068.1 ns |  5,098,568.1 ns |  1.00 |    0.01 |    21.2766 |   156646 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                              **(?i)the** |            **None** |  **1,781,944.3 ns** |    **29,817.73 ns** |    **26,432.64 ns** |  **1,775,171.5 ns** |  **1,731,885.9 ns** |  **1,819,693.7 ns** |  **1.00** |    **0.00** |   **260.5634** |  **1661305 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                              (?i)the |            None |  1,786,922.0 ns |    34,690.60 ns |    32,449.61 ns |  1,780,070.7 ns |  1,742,996.4 ns |  1,866,018.6 ns |  1.00 |    0.03 |   264.2857 |  1661302 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                              **(?i)the** |        **Compiled** |  **1,125,136.7 ns** |    **17,223.05 ns** |    **16,110.45 ns** |  **1,120,089.7 ns** |  **1,104,529.9 ns** |  **1,158,367.0 ns** |  **1.00** |    **0.00** |   **263.3929** |  **1661299 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                              (?i)the |        Compiled |  1,132,495.6 ns |    22,126.58 ns |    21,731.26 ns |  1,134,465.4 ns |  1,090,164.2 ns |  1,178,483.8 ns |  1.01 |    0.02 |   262.5000 |  1661300 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                              **(?i)the** | **NonBacktracking** |  **2,050,908.3 ns** |    **22,344.36 ns** |    **20,900.93 ns** |  **2,047,341.6 ns** |  **2,009,008.0 ns** |  **2,086,180.8 ns** |  **1.00** |    **0.00** |   **264.0000** |  **1661302 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                              (?i)the | NonBacktracking |  2,038,161.4 ns |    30,690.58 ns |    28,707.99 ns |  2,034,082.6 ns |  1,996,181.0 ns |  2,095,186.8 ns |  0.99 |    0.02 |   264.4628 |  1661302 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **(?m)^Sherlock(...)rlock Holmes$ [37]** |            **None** |     **52,808.7 ns** |       **883.36 ns** |       **826.30 ns** |     **52,758.6 ns** |     **51,047.0 ns** |     **54,159.3 ns** |  **1.00** |    **0.00** |     **1.0511** |     **7072 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | (?m)^Sherlock(...)rlock Holmes$ [37] |            None |     52,680.8 ns |       990.48 ns |       972.78 ns |     52,338.8 ns |     51,538.7 ns |     54,745.1 ns |  1.00 |    0.02 |     1.0535 |     7073 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **(?m)^Sherlock(...)rlock Holmes$ [37]** |        **Compiled** |     **47,152.1 ns** |       **465.96 ns** |       **389.10 ns** |     **47,186.9 ns** |     **46,444.3 ns** |     **47,903.5 ns** |  **1.00** |    **0.00** |     **0.9441** |     **7072 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | (?m)^Sherlock(...)rlock Holmes$ [37] |        Compiled |     47,020.5 ns |       552.95 ns |       490.17 ns |     46,950.3 ns |     46,239.7 ns |     48,062.6 ns |  1.00 |    0.01 |     0.9413 |     7072 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **(?m)^Sherlock(...)rlock Holmes$ [37]** | **NonBacktracking** |     **68,168.5 ns** |       **500.61 ns** |       **443.78 ns** |     **68,185.6 ns** |     **67,183.3 ns** |     **68,840.4 ns** |  **1.00** |    **0.00** |     **1.3591** |     **8776 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | (?m)^Sherlock(...)rlock Holmes$ [37] | NonBacktracking |     69,357.2 ns |       972.83 ns |       909.98 ns |     69,246.0 ns |     68,103.4 ns |     71,363.5 ns |  1.02 |    0.02 |     1.3423 |     8777 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **(?s).*** |            **None** |  **1,842,141.8 ns** |    **29,683.60 ns** |    **27,766.06 ns** |  **1,835,030.7 ns** |  **1,812,645.3 ns** |  **1,897,559.1 ns** |  **1.00** |    **0.00** |          **-** |      **421 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               (?s).* |            None |  1,829,645.8 ns |    17,812.88 ns |    16,662.18 ns |  1,830,611.0 ns |  1,800,585.3 ns |  1,859,569.1 ns |  0.99 |    0.01 |          - |      421 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **(?s).*** |        **Compiled** |        **162.2 ns** |         **2.87 ns** |         **2.54 ns** |        **161.6 ns** |        **158.4 ns** |        **167.1 ns** |  **1.00** |    **0.00** |     **0.0660** |      **416 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               (?s).* |        Compiled |        175.9 ns |         4.20 ns |         4.67 ns |        174.4 ns |        169.9 ns |        185.0 ns |  1.08 |    0.03 |     0.0659 |      416 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **(?s).*** | **NonBacktracking** |  **3,679,828.2 ns** |    **19,572.25 ns** |    **18,307.89 ns** |  **3,681,627.9 ns** |  **3,641,473.5 ns** |  **3,706,905.9 ns** |  **1.00** |    **0.00** |          **-** |      **427 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               (?s).* | NonBacktracking |  3,675,622.5 ns |    54,730.43 ns |    51,194.88 ns |  3,661,198.5 ns |  3,587,779.4 ns |  3,787,200.0 ns |  1.00 |    0.02 |          - |      427 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                   **.*** |            **None** |  **3,022,118.1 ns** |    **48,646.97 ns** |    **45,504.41 ns** |  **3,022,143.8 ns** |  **2,944,310.0 ns** |  **3,092,895.0 ns** |  **1.00** |    **0.00** |   **862.5000** |  **5429849 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                   .* |            None |  3,128,005.4 ns |    98,068.04 ns |   112,935.30 ns |  3,109,278.0 ns |  2,940,575.0 ns |  3,406,720.2 ns |  1.05 |    0.04 |   857.1429 |  5429849 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                   **.*** |        **Compiled** |  **2,226,955.4 ns** |    **17,472.93 ns** |    **16,344.19 ns** |  **2,226,273.2 ns** |  **2,197,458.0 ns** |  **2,252,242.0 ns** |  **1.00** |    **0.00** |   **857.1429** |  **5429847 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                   .* |        Compiled |  2,199,752.4 ns |    28,809.51 ns |    26,948.43 ns |  2,199,653.8 ns |  2,160,651.2 ns |  2,255,586.2 ns |  0.99 |    0.01 |   862.5000 |  5429849 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                   **.*** | **NonBacktracking** |  **6,331,002.6 ns** |   **105,727.02 ns** |    **98,897.12 ns** |  **6,342,592.3 ns** |  **6,142,982.1 ns** |  **6,500,833.3 ns** |  **1.00** |    **0.00** |   **846.1538** |  **5429858 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                   .* | NonBacktracking |  6,426,134.2 ns |   100,802.94 ns |    89,359.18 ns |  6,415,505.1 ns |  6,284,676.9 ns |  6,620,653.8 ns |  1.02 |    0.02 |   846.1538 |  5429858 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **Holmes** |            **None** |    **100,214.0 ns** |     **1,020.69 ns** |       **852.32 ns** |    **100,382.6 ns** |     **98,533.5 ns** |    **101,135.5 ns** |  **1.00** |    **0.00** |    **15.1329** |    **95888 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               Holmes |            None |    104,976.6 ns |     3,069.19 ns |     3,284.00 ns |    103,514.6 ns |    101,522.9 ns |    112,144.1 ns |  1.05 |    0.04 |    15.1581 |    95890 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **Holmes** |        **Compiled** |     **84,231.0 ns** |     **1,663.48 ns** |     **1,556.02 ns** |     **83,805.1 ns** |     **82,364.0 ns** |     **87,469.0 ns** |  **1.00** |    **0.00** |    **14.9601** |    **95888 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               Holmes |        Compiled |     84,440.9 ns |     1,040.09 ns |       812.03 ns |     84,227.8 ns |     83,222.1 ns |     86,488.5 ns |  1.01 |    0.01 |    15.0273 |    95888 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **Holmes** | **NonBacktracking** |    **139,301.0 ns** |     **1,442.36 ns** |     **1,204.43 ns** |    **139,156.7 ns** |    **137,737.3 ns** |    **141,349.1 ns** |  **1.00** |    **0.00** |    **14.7392** |    **95888 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               Holmes | NonBacktracking |    139,644.4 ns |     2,754.14 ns |     2,299.83 ns |    139,304.2 ns |    137,040.7 ns |    145,706.6 ns |  1.00 |    0.02 |    14.8352 |    95888 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **Holmes.{0,25}(...).{0,25}Holmes [39]** |            **None** |    **436,009.1 ns** |     **8,645.02 ns** |     **8,086.56 ns** |    **435,474.6 ns** |    **425,192.5 ns** |    **452,622.4 ns** |  **1.00** |    **0.00** |          **-** |     **1457 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | Holmes.{0,25}(...).{0,25}Holmes [39] |            None |    440,358.2 ns |    14,334.89 ns |    15,933.19 ns |    436,381.8 ns |    423,532.4 ns |    483,110.3 ns |  1.00 |    0.05 |          - |     1458 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **Holmes.{0,25}(...).{0,25}Holmes [39]** |        **Compiled** |     **90,345.2 ns** |     **1,159.89 ns** |     **1,028.21 ns** |     **89,926.7 ns** |     **89,309.2 ns** |     **92,785.2 ns** |  **1.00** |    **0.00** |          **-** |     **1456 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | Holmes.{0,25}(...).{0,25}Holmes [39] |        Compiled |     86,439.4 ns |       276.67 ns |       231.03 ns |     86,439.6 ns |     86,047.0 ns |     86,850.6 ns |  0.96 |    0.01 |          - |     1456 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **Holmes.{0,25}(...).{0,25}Holmes [39]** | **NonBacktracking** |    **199,141.0 ns** |     **1,198.55 ns** |     **1,062.49 ns** |    **198,942.9 ns** |    **197,774.8 ns** |    **201,718.1 ns** |  **1.00** |    **0.00** |          **-** |     **1457 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | Holmes.{0,25}(...).{0,25}Holmes [39] | NonBacktracking |    196,601.0 ns |       748.95 ns |       625.41 ns |    196,722.6 ns |    195,484.8 ns |    197,667.3 ns |  0.99 |    0.00 |          - |     1457 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                 **Sher[a-z]+\|Hol[a-z]+** |            **None** |    **276,662.9 ns** |     **1,601.54 ns** |     **1,419.73 ns** |    **276,767.1 ns** |    **274,260.4 ns** |    **279,214.3 ns** |  **1.00** |    **0.00** |    **18.8053** |   **121057 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                 Sher[a-z]+\|Hol[a-z]+ |            None |    274,946.7 ns |     2,290.16 ns |     1,788.01 ns |    274,512.5 ns |    272,087.4 ns |    278,609.9 ns |  0.99 |    0.01 |    18.9310 |   121057 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                 **Sher[a-z]+\|Hol[a-z]+** |        **Compiled** |    **119,058.8 ns** |       **814.01 ns** |       **679.74 ns** |    **118,809.5 ns** |    **118,277.6 ns** |    **120,493.5 ns** |  **1.00** |    **0.00** |    **19.2308** |   **121056 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                 Sher[a-z]+\|Hol[a-z]+ |        Compiled |    118,164.0 ns |     1,420.19 ns |     1,258.96 ns |    117,926.5 ns |    116,030.3 ns |    120,036.2 ns |  0.99 |    0.01 |    19.1231 |   121056 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                 **Sher[a-z]+\|Hol[a-z]+** | **NonBacktracking** |    **238,635.6 ns** |     **1,717.19 ns** |     **1,522.24 ns** |    **238,945.5 ns** |    **236,190.9 ns** |    **241,160.2 ns** |  **1.00** |    **0.00** |    **18.8679** |   **121057 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                 Sher[a-z]+\|Hol[a-z]+ | NonBacktracking |    239,829.8 ns |     1,430.23 ns |     1,267.86 ns |    239,825.8 ns |    237,623.1 ns |    242,214.6 ns |  1.01 |    0.01 |    18.9394 |   121057 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                             **Sherlock** |            **None** |     **54,094.8 ns** |       **707.75 ns** |       **662.03 ns** |     **53,958.9 ns** |     **53,264.4 ns** |     **55,246.2 ns** |  **1.00** |    **0.00** |     **3.0082** |    **20176 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                             Sherlock |            None |     54,334.8 ns |       426.71 ns |       356.32 ns |     54,281.2 ns |     53,845.4 ns |     55,220.3 ns |  1.00 |    0.01 |     3.0395 |    20176 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                             **Sherlock** |        **Compiled** |     **49,506.1 ns** |       **374.28 ns** |       **331.79 ns** |     **49,435.7 ns** |     **49,009.4 ns** |     **50,129.3 ns** |  **1.00** |    **0.00** |     **3.1250** |    **20176 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                             Sherlock |        Compiled |     49,966.9 ns |       303.80 ns |       284.17 ns |     49,949.2 ns |     49,350.8 ns |     50,436.6 ns |  1.01 |    0.01 |     3.1847 |    20176 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                             **Sherlock** | **NonBacktracking** |     **65,026.2 ns** |       **577.84 ns** |       **482.52 ns** |     **64,813.1 ns** |     **64,476.9 ns** |     **65,953.0 ns** |  **1.00** |    **0.00** |     **3.1217** |    **20176 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                             Sherlock | NonBacktracking |     65,319.3 ns |       642.16 ns |       600.68 ns |     65,329.4 ns |     64,570.6 ns |     66,236.8 ns |  1.01 |    0.01 |     3.1266 |    20176 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock Holmes** |            **None** |     **54,059.8 ns** |       **530.53 ns** |       **470.30 ns** |     **54,048.9 ns** |     **53,294.2 ns** |     **54,942.0 ns** |  **1.00** |    **0.00** |     **2.8072** |    **18929 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock Holmes |            None |     54,533.6 ns |       792.18 ns |       702.25 ns |     54,329.5 ns |     53,739.2 ns |     55,844.6 ns |  1.01 |    0.02 |     2.8041 |    18928 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock Holmes** |        **Compiled** |     **49,776.7 ns** |       **414.76 ns** |       **387.97 ns** |     **49,703.4 ns** |     **49,250.4 ns** |     **50,604.8 ns** |  **1.00** |    **0.00** |     **2.8317** |    **18928 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock Holmes |        Compiled |     49,700.4 ns |       429.07 ns |       401.36 ns |     49,598.1 ns |     49,126.7 ns |     50,625.4 ns |  1.00 |    0.01 |     2.9857 |    18928 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock Holmes** | **NonBacktracking** |     **76,513.4 ns** |       **885.18 ns** |       **827.99 ns** |     **76,309.4 ns** |     **75,320.8 ns** |     **77,848.7 ns** |  **1.00** |    **0.00** |     **2.7414** |    **18928 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock Holmes | NonBacktracking |     75,706.8 ns |       551.24 ns |       515.63 ns |     75,647.6 ns |     74,874.3 ns |     76,563.2 ns |  0.99 |    0.01 |     2.7481 |    18928 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                    **Sherlock\s+Holmes** |            **None** |     **58,239.4 ns** |       **971.58 ns** |       **997.75 ns** |     **57,870.4 ns** |     **56,906.7 ns** |     **60,902.4 ns** |  **1.00** |    **0.00** |     **2.9947** |    **20176 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                    Sherlock\s+Holmes |            None |     56,973.5 ns |       639.39 ns |       566.81 ns |     56,730.2 ns |     56,369.2 ns |     58,258.9 ns |  0.98 |    0.02 |     3.1782 |    20176 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                    **Sherlock\s+Holmes** |        **Compiled** |     **51,490.1 ns** |       **701.61 ns** |       **656.28 ns** |     **51,227.8 ns** |     **50,501.7 ns** |     **52,644.2 ns** |  **1.00** |    **0.00** |     **3.0438** |    **20176 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                    Sherlock\s+Holmes |        Compiled |     51,072.9 ns |       406.51 ns |       360.36 ns |     50,925.6 ns |     50,682.1 ns |     51,709.5 ns |  0.99 |    0.01 |     3.0340 |    20176 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                    **Sherlock\s+Holmes** | **NonBacktracking** |     **93,653.4 ns** |       **929.00 ns** |       **823.54 ns** |     **93,750.8 ns** |     **92,135.1 ns** |     **94,721.2 ns** |  **1.00** |    **0.00** |     **2.9586** |    **20176 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                    Sherlock\s+Holmes | NonBacktracking |     94,019.5 ns |     1,864.97 ns |     1,915.18 ns |     93,448.6 ns |     91,773.9 ns |     98,065.0 ns |  1.01 |    0.02 |     2.9564 |    20177 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock\|Holmes** |            **None** |    **263,251.6 ns** |     **4,467.83 ns** |     **3,960.61 ns** |    **262,462.8 ns** |    **259,013.5 ns** |    **273,370.2 ns** |  **1.00** |    **0.00** |    **17.9136** |   **116065 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock\|Holmes |            None |    262,733.4 ns |     2,643.76 ns |     2,472.97 ns |    262,899.2 ns |    259,356.2 ns |    267,154.4 ns |  1.00 |    0.01 |    17.8571 |   116065 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock\|Holmes** |        **Compiled** |    **113,271.6 ns** |       **517.82 ns** |       **404.28 ns** |    **113,350.3 ns** |    **112,588.7 ns** |    **113,877.7 ns** |  **1.00** |    **0.00** |    **18.3824** |   **116064 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock\|Holmes |        Compiled |    112,620.3 ns |       443.77 ns |       346.47 ns |    112,637.4 ns |    111,845.5 ns |    113,035.1 ns |  0.99 |    0.01 |    18.4353 |   116064 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock\|Holmes** | **NonBacktracking** |    **240,908.2 ns** |     **1,753.24 ns** |     **1,639.98 ns** |    **240,742.5 ns** |    **238,892.1 ns** |    **243,424.6 ns** |  **1.00** |    **0.00** |    **18.3575** |   **116065 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock\|Holmes | NonBacktracking |    249,661.8 ns |     7,329.38 ns |     8,146.59 ns |    245,970.1 ns |    242,789.1 ns |    266,959.8 ns |  1.04 |    0.04 |    18.4287 |   116065 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |               **Sherlock\|Holmes\|Watson** |            **None** |    **424,356.2 ns** |     **5,703.88 ns** |     **5,857.47 ns** |    **423,176.4 ns** |    **418,034.4 ns** |    **438,508.0 ns** |  **1.00** |    **0.00** |    **19.9275** |   **132919 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |               Sherlock\|Holmes\|Watson |            None |    412,588.3 ns |     6,338.19 ns |     5,618.64 ns |    411,696.0 ns |    405,914.9 ns |    427,364.5 ns |  0.97 |    0.02 |    19.6399 |   132913 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |               **Sherlock\|Holmes\|Watson** |        **Compiled** |    **157,810.1 ns** |     **1,325.70 ns** |     **1,175.20 ns** |    **157,210.9 ns** |    **156,222.3 ns** |    **159,680.0 ns** |  **1.00** |    **0.00** |    **20.8333** |   **132912 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |               Sherlock\|Holmes\|Watson |        Compiled |    151,499.5 ns |       844.22 ns |       748.37 ns |    151,237.0 ns |    150,621.1 ns |    152,757.4 ns |  0.96 |    0.01 |    21.0337 |   132912 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |               **Sherlock\|Holmes\|Watson** | **NonBacktracking** |    **312,895.5 ns** |     **2,983.18 ns** |     **2,644.51 ns** |    **312,976.4 ns** |    **309,332.2 ns** |    **317,511.9 ns** |  **1.00** |    **0.00** |    **20.0000** |   **132913 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |               Sherlock\|Holmes\|Watson | NonBacktracking |    305,635.9 ns |     3,798.06 ns |     3,552.71 ns |    304,460.0 ns |    301,111.1 ns |    312,056.8 ns |  0.98 |    0.01 |    20.5811 |   132916 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **Sherlock\|Holm(...)er\|John\|Baker [45]** |            **None** |  **2,434,333.5 ns** |     **9,936.46 ns** |     **7,757.73 ns** |  **2,432,304.5 ns** |  **2,424,309.9 ns** |  **2,449,734.7 ns** |  **1.00** |    **0.00** |    **19.8020** |   **153927 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | Sherlock\|Holm(...)er\|John\|Baker [45] |            None |  2,402,548.4 ns |    15,276.84 ns |    12,756.85 ns |  2,399,813.5 ns |  2,385,050.0 ns |  2,425,371.2 ns |  0.99 |    0.01 |    19.2308 |   153927 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **Sherlock\|Holm(...)er\|John\|Baker [45]** |        **Compiled** |  **2,055,268.1 ns** |    **19,810.10 ns** |    **17,561.14 ns** |  **2,063,414.5 ns** |  **2,024,755.5 ns** |  **2,076,677.3 ns** |  **1.00** |    **0.00** |    **23.4375** |   **153926 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | Sherlock\|Holm(...)er\|John\|Baker [45] |        Compiled |  2,029,780.1 ns |     6,745.40 ns |     5,979.63 ns |  2,028,710.5 ns |  2,021,491.4 ns |  2,043,643.0 ns |  0.99 |    0.01 |    23.4375 |   153926 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** | **Sherlock\|Holm(...)er\|John\|Baker [45]** | **NonBacktracking** |  **3,006,329.4 ns** |     **9,536.74 ns** |     **8,454.07 ns** |  **3,003,704.9 ns** |  **2,993,019.5 ns** |  **3,020,035.4 ns** |  **1.00** |    **0.00** |    **24.3902** |   **153936 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe | Sherlock\|Holm(...)er\|John\|Baker [45] | NonBacktracking |  2,961,733.4 ns |    16,191.51 ns |    13,520.64 ns |  2,958,759.0 ns |  2,938,394.0 ns |  2,994,372.3 ns |  0.99 |    0.01 |    24.0964 |   153929 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock\|Street** |            **None** |    **126,457.3 ns** |     **2,525.33 ns** |     **2,806.90 ns** |    **126,160.6 ns** |    **122,640.2 ns** |    **132,041.5 ns** |  **1.00** |    **0.00** |     **5.1733** |    **32866 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock\|Street |            None |    126,518.9 ns |     3,499.75 ns |     3,744.70 ns |    126,125.6 ns |    121,766.4 ns |    135,403.5 ns |  1.00 |    0.04 |     5.0302 |    32865 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock\|Street** |        **Compiled** |     **57,117.6 ns** |       **659.72 ns** |       **617.10 ns** |     **57,287.6 ns** |     **55,862.8 ns** |     **57,955.8 ns** |  **1.00** |    **0.00** |     **5.0738** |    **32864 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock\|Street |        Compiled |     57,343.9 ns |       677.07 ns |       600.21 ns |     57,580.2 ns |     56,208.0 ns |     58,131.0 ns |  1.00 |    0.02 |     5.0551 |    32864 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                      **Sherlock\|Street** | **NonBacktracking** |    **106,663.2 ns** |       **934.82 ns** |       **874.43 ns** |    **106,411.5 ns** |    **105,543.1 ns** |    **108,650.3 ns** |  **1.00** |    **0.00** |     **5.0590** |    **32864 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                      Sherlock\|Street | NonBacktracking |    106,429.2 ns |     1,309.81 ns |     1,225.20 ns |    106,070.6 ns |    104,865.4 ns |    109,043.0 ns |  1.00 |    0.01 |     5.0272 |    32864 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **The** |            **None** |    **145,622.3 ns** |    **12,600.59 ns** |    **14,510.85 ns** |    **138,516.5 ns** |    **132,179.2 ns** |    **174,355.5 ns** |  **1.00** |    **0.00** |    **24.4965** |   **154129 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  The |            None |    137,432.6 ns |     2,860.71 ns |     3,294.40 ns |    136,836.1 ns |    133,528.1 ns |    143,523.5 ns |  0.95 |    0.09 |    24.1096 |   154130 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **The** |        **Compiled** |    **110,164.8 ns** |     **2,143.97 ns** |     **2,469.00 ns** |    **109,476.3 ns** |    **106,674.1 ns** |    **115,753.3 ns** |  **1.00** |    **0.00** |    **24.3056** |   **154128 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  The |        Compiled |    107,741.4 ns |       821.67 ns |       686.14 ns |    107,736.5 ns |    106,607.3 ns |    109,098.6 ns |  0.98 |    0.02 |    24.3056 |   154128 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **The** | **NonBacktracking** |    **166,464.2 ns** |     **1,452.56 ns** |     **1,287.66 ns** |    **166,198.8 ns** |    **164,252.4 ns** |    **168,901.0 ns** |  **1.00** |    **0.00** |    **24.2464** |   **154130 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  The | NonBacktracking |    165,199.3 ns |     1,382.84 ns |     1,293.51 ns |    164,805.7 ns |    163,278.1 ns |    167,522.6 ns |  0.99 |    0.01 |    24.4386 |   154128 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **[^\n]*** |            **None** |  **3,257,966.5 ns** |   **157,547.50 ns** |   **181,431.94 ns** |  **3,206,710.4 ns** |  **3,077,167.1 ns** |  **3,577,608.5 ns** |  **1.00** |    **0.00** |   **853.6585** |  **5429849 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               [^\n]* |            None |  3,223,109.5 ns |    61,368.18 ns |    57,403.83 ns |  3,229,860.3 ns |  3,126,701.3 ns |  3,354,802.6 ns |  1.01 |    0.04 |   858.9744 |  5429886 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **[^\n]*** |        **Compiled** |  **2,301,864.0 ns** |    **39,436.36 ns** |    **36,888.79 ns** |  **2,285,650.0 ns** |  **2,244,072.3 ns** |  **2,366,262.5 ns** |  **1.00** |    **0.00** |   **857.1429** |  **5429847 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               [^\n]* |        Compiled |  2,259,107.4 ns |    17,473.11 ns |    13,641.85 ns |  2,255,550.9 ns |  2,244,500.9 ns |  2,291,052.7 ns |  0.98 |    0.02 |   857.1429 |  5429847 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **[^\n]*** | **NonBacktracking** |  **6,415,712.6 ns** |    **28,040.18 ns** |    **24,856.89 ns** |  **6,417,724.4 ns** |  **6,376,769.2 ns** |  **6,474,200.0 ns** |  **1.00** |    **0.00** |   **846.1538** |  **5429858 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               [^\n]* | NonBacktracking |  6,557,223.6 ns |   129,939.10 ns |   127,617.57 ns |  6,531,655.1 ns |  6,418,343.6 ns |  6,844,912.8 ns |  1.02 |    0.02 |   846.1538 |  5429858 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                     **[a-q][^u-z]{13}x** |            **None** |     **77,618.1 ns** |     **1,541.43 ns** |     **1,441.86 ns** |     **77,031.2 ns** |     **76,009.6 ns** |     **80,394.0 ns** |  **1.00** |    **0.00** |     **4.5372** |    **29537 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                     [a-q][^u-z]{13}x |            None |     79,057.5 ns |     1,164.02 ns |       908.79 ns |     79,243.8 ns |     77,542.7 ns |     80,305.6 ns |  1.02 |    0.03 |     4.6067 |    29536 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                     **[a-q][^u-z]{13}x** |        **Compiled** |     **49,965.6 ns** |       **869.76 ns** |       **771.02 ns** |     **49,884.7 ns** |     **48,875.5 ns** |     **51,223.6 ns** |  **1.00** |    **0.00** |     **4.5635** |    **29536 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                     [a-q][^u-z]{13}x |        Compiled |     49,914.1 ns |       629.00 ns |       557.59 ns |     49,837.8 ns |     49,200.4 ns |     51,320.4 ns |  1.00 |    0.02 |     4.5780 |    29536 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                     **[a-q][^u-z]{13}x** | **NonBacktracking** |    **121,485.8 ns** |     **1,347.44 ns** |     **1,194.47 ns** |    **121,402.6 ns** |    **119,673.1 ns** |    **123,787.5 ns** |  **1.00** |    **0.00** |     **4.3732** |    **29536 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                     [a-q][^u-z]{13}x | NonBacktracking |    124,021.8 ns |     1,798.04 ns |     1,681.89 ns |    123,936.5 ns |    121,400.5 ns |    127,968.3 ns |  1.02 |    0.02 |     4.3520 |    29536 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **[a-zA-Z]+ing** |            **None** | **17,501,495.5 ns** |   **315,411.28 ns** |   **279,603.88 ns** | **17,426,581.8 ns** | **17,133,890.9 ns** | **17,961,209.1 ns** |  **1.00** |    **0.00** |    **90.9091** |   **587457 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         [a-zA-Z]+ing |            None | 17,303,185.0 ns |   349,899.53 ns |   402,944.82 ns | 17,151,209.1 ns | 16,705,609.1 ns | 18,254,718.2 ns |  1.00 |    0.03 |    90.9091 |   587457 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **[a-zA-Z]+ing** |        **Compiled** |  **4,852,160.4 ns** |    **84,033.53 ns** |    **82,532.17 ns** |  **4,815,328.9 ns** |  **4,776,464.1 ns** |  **5,030,210.9 ns** |  **1.00** |    **0.00** |    **78.1250** |   **587406 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         [a-zA-Z]+ing |        Compiled |  4,874,588.2 ns |    75,242.30 ns |    66,700.34 ns |  4,860,221.1 ns |  4,800,292.2 ns |  5,035,029.7 ns |  1.01 |    0.02 |    78.1250 |   587404 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **[a-zA-Z]+ing** | **NonBacktracking** |  **7,979,291.9 ns** |    **38,559.51 ns** |    **34,182.00 ns** |  **7,974,526.7 ns** |  **7,942,080.0 ns** |  **8,062,256.7 ns** |  **1.00** |    **0.00** |    **66.6667** |   **587416 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         [a-zA-Z]+ing | NonBacktracking |  7,878,343.1 ns |    21,973.90 ns |    20,554.40 ns |  7,879,830.0 ns |  7,836,553.3 ns |  7,913,870.0 ns |  0.99 |    0.00 |    66.6667 |   587416 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                             **\b\w+n\b** |            **None** | **31,505,205.4 ns** |   **299,046.85 ns** |   **265,097.25 ns** | **31,362,087.5 ns** | **31,224,200.0 ns** | **32,068,437.5 ns** |  **1.00** |    **0.00** |   **250.0000** |  **1740218 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                             \b\w+n\b |            None | 31,849,141.1 ns |   579,426.42 ns |   513,646.43 ns | 31,695,387.5 ns | 31,214,762.5 ns | 32,929,612.5 ns |  1.01 |    0.02 |   250.0000 |  1740218 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                             **\b\w+n\b** |        **Compiled** | **11,003,567.9 ns** |    **48,556.62 ns** |    **43,044.18 ns** | **11,006,893.8 ns** | **10,930,962.5 ns** | **11,079,850.0 ns** |  **1.00** |    **0.00** |   **250.0000** |  **1740218 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                             \b\w+n\b |        Compiled | 11,508,970.1 ns |    28,677.71 ns |    25,422.05 ns | 11,510,527.3 ns | 11,468,113.6 ns | 11,552,509.1 ns |  1.05 |    0.00 |   272.7273 |  1740161 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                             **\b\w+n\b** | **NonBacktracking** | **10,001,656.3 ns** |    **30,702.94 ns** |    **25,638.34 ns** | **10,004,972.0 ns** |  **9,963,472.0 ns** | **10,057,180.0 ns** |  **1.00** |    **0.00** |   **320.0000** |  **2097541 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                             \b\w+n\b | NonBacktracking | 10,265,520.5 ns |   142,506.71 ns |   146,343.90 ns | 10,212,836.0 ns | 10,112,548.0 ns | 10,637,532.0 ns |  1.03 |    0.02 |   320.0000 |  2097541 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **\p{Ll}** |            **None** | **47,725,678.6 ns** |   **694,907.20 ns** |   **616,017.13 ns** | **47,516,837.5 ns** | **47,065,775.0 ns** | **49,205,975.0 ns** |  **1.00** |    **0.00** | **14250.0000** | **90060020 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               \p{Ll} |            None | 48,601,967.9 ns |   581,532.54 ns |   515,513.45 ns | 48,641,762.5 ns | 47,645,200.0 ns | 49,511,625.0 ns |  1.02 |    0.02 | 14250.0000 | 90060020 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **\p{Ll}** |        **Compiled** | **33,347,741.7 ns** |   **479,227.66 ns** |   **448,269.85 ns** | **33,170,400.0 ns** | **32,781,850.0 ns** | **34,325,850.0 ns** |  **1.00** |    **0.00** | **14250.0000** | **90060020 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               \p{Ll} |        Compiled | 34,567,153.9 ns |   676,024.66 ns |   751,399.62 ns | 34,328,425.0 ns | 33,496,350.0 ns | 36,065,050.0 ns |  1.04 |    0.02 | 14250.0000 | 90060020 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **\p{Ll}** | **NonBacktracking** | **58,653,236.7 ns** |   **675,861.41 ns** |   **632,201.18 ns** | **58,636,775.0 ns** | **57,579,175.0 ns** | **59,563,400.0 ns** |  **1.00** |    **0.00** | **14250.0000** | **90060020 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               \p{Ll} | NonBacktracking | 59,420,033.9 ns |   912,459.01 ns |   808,871.15 ns | 59,564,337.5 ns | 58,322,200.0 ns | 60,811,925.0 ns |  1.01 |    0.02 | 14250.0000 | 90060202 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **\p{Lu}** |            **None** |  **2,575,557.9 ns** |    **29,311.71 ns** |    **24,476.60 ns** |  **2,572,980.2 ns** |  **2,536,355.2 ns** |  **2,631,352.1 ns** |  **1.00** |    **0.00** |   **468.7500** |  **2949454 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               \p{Lu} |            None |  2,608,828.1 ns |    24,084.23 ns |    20,111.42 ns |  2,605,348.0 ns |  2,576,504.1 ns |  2,650,918.4 ns |  1.01 |    0.02 |   469.3878 |  2949447 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **\p{Lu}** |        **Compiled** |  **1,760,491.3 ns** |    **28,596.77 ns** |    **23,879.59 ns** |  **1,761,091.1 ns** |  **1,711,475.0 ns** |  **1,790,882.1 ns** |  **1.00** |    **0.00** |   **464.2857** |  **2949446 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               \p{Lu} |        Compiled |  1,741,510.9 ns |    13,423.72 ns |    11,209.41 ns |  1,742,343.8 ns |  1,727,103.5 ns |  1,768,081.2 ns |  0.99 |    0.01 |   465.2778 |  2949445 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                               **\p{Lu}** | **NonBacktracking** |  **2,925,336.7 ns** |    **23,347.79 ns** |    **20,697.21 ns** |  **2,926,228.6 ns** |  **2,893,525.0 ns** |  **2,965,329.8 ns** |  **1.00** |    **0.00** |   **464.2857** |  **2949449 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                               \p{Lu} | NonBacktracking |  2,922,340.1 ns |    22,728.58 ns |    21,260.33 ns |  2,918,580.2 ns |  2,884,581.4 ns |  2,962,097.7 ns |  1.00 |    0.01 |   465.1163 |  2949448 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                **\p{L}** |            **None** | **48,617,098.1 ns** |   **412,806.21 ns** |   **344,711.88 ns** | **48,600,300.0 ns** | **48,178,225.0 ns** | **49,325,750.0 ns** |  **1.00** |    **0.00** | **14750.0000** | **93009460 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                \p{L} |            None | 51,349,297.5 ns | 1,694,399.29 ns | 1,951,272.75 ns | 51,208,150.0 ns | 48,944,450.0 ns | 55,910,400.0 ns |  1.04 |    0.04 | 14750.0000 | 93009460 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                **\p{L}** |        **Compiled** | **34,696,382.8 ns** |   **675,664.03 ns** |   **663,592.42 ns** | **34,539,500.0 ns** | **33,854,900.0 ns** | **36,060,950.0 ns** |  **1.00** |    **0.00** | **14750.0000** | **93009460 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                \p{L} |        Compiled | 34,442,605.8 ns |   343,787.45 ns |   287,078.09 ns | 34,467,900.0 ns | 33,792,050.0 ns | 34,868,950.0 ns |  0.99 |    0.02 | 14750.0000 | 93009460 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                **\p{L}** | **NonBacktracking** | **59,876,165.3 ns** | **1,064,857.13 ns** | **1,139,385.19 ns** | **59,545,350.0 ns** | **58,566,750.0 ns** | **62,547,650.0 ns** |  **1.00** |    **0.00** | **14750.0000** | **93009460 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                \p{L} | NonBacktracking | 61,502,975.0 ns | 1,258,160.08 ns | 1,346,217.19 ns | 61,972,162.5 ns | 58,561,250.0 ns | 63,560,725.0 ns |  1.03 |    0.02 | 14750.0000 | 93009460 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                **\s[a-zA-Z]{0,12}ing\s** |            **None** | **20,494,795.6 ns** |   **556,337.32 ns** |   **595,274.70 ns** | **20,559,685.0 ns** | **19,516,200.0 ns** | **21,472,390.0 ns** |  **1.00** |    **0.00** |          **-** |   **432920 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                \s[a-zA-Z]{0,12}ing\s |            None | 19,331,623.6 ns |   124,469.65 ns |   110,339.10 ns | 19,317,935.0 ns | 19,205,000.0 ns | 19,542,770.0 ns |  0.94 |    0.03 |          - |   432920 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                **\s[a-zA-Z]{0,12}ing\s** |        **Compiled** |  **5,817,803.0 ns** |    **33,415.48 ns** |    **29,621.95 ns** |  **5,813,009.4 ns** |  **5,781,527.1 ns** |  **5,879,329.2 ns** |  **1.00** |    **0.00** |    **62.5000** |   **432864 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                \s[a-zA-Z]{0,12}ing\s |        Compiled |  5,799,410.0 ns |    28,402.74 ns |    26,567.94 ns |  5,786,639.6 ns |  5,768,135.4 ns |  5,851,854.2 ns |  1.00 |    0.01 |    62.5000 |   432864 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                **\s[a-zA-Z]{0,12}ing\s** | **NonBacktracking** |  **5,007,168.9 ns** |    **59,859.18 ns** |    **55,992.31 ns** |  **4,995,764.6 ns** |  **4,938,352.1 ns** |  **5,148,877.1 ns** |  **1.00** |    **0.00** |    **62.5000** |   **432863 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                \s[a-zA-Z]{0,12}ing\s | NonBacktracking |  4,994,500.6 ns |    23,643.49 ns |    20,959.33 ns |  4,994,642.2 ns |  4,962,970.6 ns |  5,035,202.0 ns |  1.00 |    0.01 |    58.8235 |   432874 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **\w+** |            **None** | **15,467,008.9 ns** |    **77,982.52 ns** |    **69,129.48 ns** | **15,453,241.7 ns** | **15,347,125.0 ns** | **15,589,766.7 ns** |  **1.00** |    **0.00** |  **3583.3333** | **22716572 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  \w+ |            None | 16,034,399.6 ns |   316,205.43 ns |   280,307.88 ns | 15,916,075.0 ns | 15,717,868.8 ns | 16,724,050.0 ns |  1.04 |    0.02 |  3562.5000 | 22716557 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **\w+** |        **Compiled** | **11,379,387.8 ns** |   **160,660.50 ns** |   **150,281.93 ns** | **11,419,108.7 ns** | **11,107,495.7 ns** | **11,585,565.2 ns** |  **1.00** |    **0.00** |  **3608.6957** | **22716575 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  \w+ |        Compiled | 11,451,393.0 ns |   145,356.41 ns |   135,966.48 ns | 11,467,923.8 ns | 11,177,047.6 ns | 11,711,800.0 ns |  1.01 |    0.01 |  3619.0476 | 22716546 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **\w+** | **NonBacktracking** | **21,396,108.1 ns** |   **153,348.66 ns** |   **143,442.43 ns** | **21,373,800.0 ns** | **21,143,122.2 ns** | **21,598,033.3 ns** |  **1.00** |    **0.00** |  **3555.5556** | **22716592 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  \w+ | NonBacktracking | 21,677,960.1 ns |   104,437.81 ns |    87,210.30 ns | 21,689,145.5 ns | 21,495,927.3 ns | 21,765,954.5 ns |  1.01 |    0.01 |  3545.4545 | 22716577 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **\w+\s+Holmes** |            **None** | **11,221,374.6 ns** |   **105,095.45 ns** |    **93,164.38 ns** | **11,193,455.0 ns** | **11,070,600.0 ns** | **11,362,205.0 ns** |  **1.00** |    **0.00** |          **-** |    **66424 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         \w+\s+Holmes |            None | 11,768,162.6 ns |   487,236.26 ns |   541,561.81 ns | 11,559,440.0 ns | 11,226,480.0 ns | 13,007,060.0 ns |  1.05 |    0.05 |          - |    66388 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **\w+\s+Holmes** |        **Compiled** |  **4,024,637.6 ns** |    **89,289.46 ns** |    **91,693.70 ns** |  **4,039,987.5 ns** |  **3,919,262.5 ns** |  **4,236,945.3 ns** |  **1.00** |    **0.00** |          **-** |    **66363 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         \w+\s+Holmes |        Compiled |  4,039,828.3 ns |    43,707.87 ns |    38,745.89 ns |  4,041,342.2 ns |  3,984,274.2 ns |  4,111,833.6 ns |  1.00 |    0.02 |          - |    66364 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                         **\w+\s+Holmes** | **NonBacktracking** |  **4,275,510.0 ns** |    **31,474.46 ns** |    **29,441.23 ns** |  **4,262,479.3 ns** |  **4,239,684.5 ns** |  **4,339,991.4 ns** |  **1.00** |    **0.00** |          **-** |    **66374 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                         \w+\s+Holmes | NonBacktracking |  4,579,980.2 ns |   401,294.63 ns |   446,037.92 ns |  4,360,389.5 ns |  4,236,535.1 ns |  5,573,508.8 ns |  1.07 |    0.11 |          - |    66365 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                   **\w+\s+Holmes\s+\w+** |            **None** | **11,451,783.2 ns** |   **328,111.36 ns** |   **364,694.91 ns** | **11,315,695.2 ns** | **11,048,619.0 ns** | **12,459,909.5 ns** |  **1.00** |    **0.00** |          **-** |    **28530 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                   \w+\s+Holmes\s+\w+ |            None | 11,196,986.1 ns |   143,113.78 ns |   126,866.63 ns | 11,173,931.0 ns | 11,038,100.0 ns | 11,501,571.4 ns |  0.98 |    0.04 |          - |    28530 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                   **\w+\s+Holmes\s+\w+** |        **Compiled** |  **3,905,598.6 ns** |    **30,448.24 ns** |    **28,481.31 ns** |  **3,915,379.7 ns** |  **3,839,906.2 ns** |  **3,954,606.2 ns** |  **1.00** |    **0.00** |          **-** |    **28508 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                   \w+\s+Holmes\s+\w+ |        Compiled |  3,984,323.2 ns |    31,199.19 ns |    26,052.74 ns |  3,978,759.4 ns |  3,950,715.6 ns |  4,050,535.9 ns |  1.02 |    0.01 |          - |    28508 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                   **\w+\s+Holmes\s+\w+** | **NonBacktracking** |  **4,281,476.2 ns** |    **54,463.02 ns** |    **48,280.05 ns** |  **4,300,917.5 ns** |  **4,208,859.6 ns** |  **4,353,496.5 ns** |  **1.00** |    **0.00** |          **-** |    **28509 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                   \w+\s+Holmes\s+\w+ | NonBacktracking |  4,213,251.0 ns |    56,975.99 ns |    44,483.11 ns |  4,222,100.0 ns |  4,128,650.0 ns |  4,307,956.7 ns |  0.98 |    0.02 |          - |    28508 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **aei** |            **None** |     **54,131.3 ns** |       **475.84 ns** |       **421.82 ns** |     **54,128.9 ns** |     **53,293.1 ns** |     **54,766.5 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  aei |            None |     54,249.4 ns |       433.71 ns |       405.69 ns |     54,218.5 ns |     53,622.4 ns |     54,981.0 ns |  1.00 |    0.01 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **aei** |        **Compiled** |     **54,245.1 ns** |       **762.98 ns** |       **713.70 ns** |     **53,955.8 ns** |     **53,337.5 ns** |     **55,734.2 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  aei |        Compiled |     54,380.4 ns |       374.61 ns |       332.09 ns |     54,401.5 ns |     53,914.5 ns |     54,914.7 ns |  1.00 |    0.01 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **aei** | **NonBacktracking** |     **52,077.0 ns** |       **844.30 ns** |       **789.76 ns** |     **51,989.6 ns** |     **51,153.4 ns** |     **53,374.4 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  aei | NonBacktracking |     51,722.6 ns |       544.76 ns |       509.57 ns |     51,776.9 ns |     50,978.6 ns |     52,802.9 ns |  0.99 |    0.02 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **aqj** |            **None** |     **39,192.6 ns** |       **475.55 ns** |       **421.56 ns** |     **39,178.1 ns** |     **38,620.7 ns** |     **39,929.3 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  aqj |            None |     38,907.8 ns |       337.39 ns |       315.59 ns |     38,956.6 ns |     38,398.3 ns |     39,377.8 ns |  0.99 |    0.01 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **aqj** |        **Compiled** |     **39,192.3 ns** |       **292.74 ns** |       **273.83 ns** |     **39,168.6 ns** |     **38,678.7 ns** |     **39,656.4 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  aqj |        Compiled |     39,365.5 ns |       516.89 ns |       483.50 ns |     39,432.5 ns |     38,384.9 ns |     40,269.8 ns |  1.00 |    0.01 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **aqj** | **NonBacktracking** |     **37,404.4 ns** |       **542.58 ns** |       **480.98 ns** |     **37,485.6 ns** |     **36,627.0 ns** |     **38,253.3 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  aqj | NonBacktracking |     36,956.3 ns |       243.81 ns |       190.35 ns |     36,956.8 ns |     36,695.5 ns |     37,174.2 ns |  0.99 |    0.01 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **the** |            **None** |    **927,773.3 ns** |     **5,512.90 ns** |     **4,603.52 ns** |    **928,049.8 ns** |    **922,185.8 ns** |    **938,539.0 ns** |  **1.00** |    **0.00** |   **235.9551** |  **1501347 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  the |            None |    946,328.8 ns |    18,762.57 ns |    18,427.35 ns |    947,356.1 ns |    917,767.7 ns |    973,977.6 ns |  1.02 |    0.02 |   235.7414 |  1501354 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **the** |        **Compiled** |    **697,223.1 ns** |     **6,090.05 ns** |     **5,398.67 ns** |    **696,592.6 ns** |    **689,306.5 ns** |    **706,804.0 ns** |  **1.00** |    **0.00** |   **238.6364** |  **1501346 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  the |        Compiled |    706,496.9 ns |    12,492.76 ns |    13,367.11 ns |    704,742.8 ns |    689,116.2 ns |    737,644.3 ns |  1.01 |    0.02 |   238.6364 |  1501346 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **the** | **NonBacktracking** |  **1,261,810.0 ns** |    **16,973.08 ns** |    **15,876.63 ns** |  **1,261,002.0 ns** |  **1,242,233.8 ns** |  **1,300,236.4 ns** |  **1.00** |    **0.00** |   **237.3737** |  **1501348 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  the | NonBacktracking |  1,249,491.2 ns |    21,750.72 ns |    19,281.45 ns |  1,250,393.5 ns |  1,218,571.0 ns |  1,290,048.0 ns |  0.99 |    0.01 |   235.0000 |  1501348 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                            **the\s+\w+** |            **None** |  **1,194,728.2 ns** |    **22,518.55 ns** |    **22,116.23 ns** |  **1,187,979.3 ns** |  **1,165,817.4 ns** |  **1,245,370.9 ns** |  **1.00** |    **0.00** |   **178.4038** |  **1125296 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                            the\s+\w+ |            None |  1,205,642.6 ns |    15,771.18 ns |    13,169.65 ns |  1,204,124.6 ns |  1,175,400.0 ns |  1,229,692.3 ns |  1.01 |    0.02 |   178.7440 |  1125283 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                            **the\s+\w+** |        **Compiled** |    **785,689.9 ns** |     **8,981.74 ns** |     **7,962.08 ns** |    **786,087.4 ns** |    **771,909.8 ns** |    **800,795.5 ns** |  **1.00** |    **0.00** |   **178.5714** |  **1125282 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                            the\s+\w+ |        Compiled |    812,239.1 ns |    10,937.36 ns |    10,230.81 ns |    810,591.2 ns |    800,685.0 ns |    833,347.5 ns |  1.03 |    0.02 |   178.1250 |  1125282 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                            **the\s+\w+** | **NonBacktracking** |  **1,715,321.8 ns** |    **21,632.75 ns** |    **19,176.87 ns** |  **1,717,738.5 ns** |  **1,686,959.4 ns** |  **1,745,328.0 ns** |  **1.00** |    **0.00** |   **174.8252** |  **1125289 B** |        **1.00** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                            the\s+\w+ | NonBacktracking |  1,770,189.8 ns |    31,008.82 ns |    29,005.67 ns |  1,770,205.0 ns |  1,728,962.1 ns |  1,836,358.6 ns |  1.03 |    0.02 |   178.5714 |  1125285 B |        1.00 |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **zqj** |            **None** |     **39,110.9 ns** |       **331.99 ns** |       **277.22 ns** |     **39,159.9 ns** |     **38,736.7 ns** |     **39,609.7 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  zqj |            None |     40,627.1 ns |       526.74 ns |       492.71 ns |     40,470.9 ns |     39,984.2 ns |     41,598.4 ns |  1.04 |    0.02 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **zqj** |        **Compiled** |     **39,362.2 ns** |       **418.72 ns** |       **391.67 ns** |     **39,328.2 ns** |     **38,684.4 ns** |     **39,989.8 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  zqj |        Compiled |     39,205.9 ns |       417.27 ns |       369.90 ns |     39,120.6 ns |     38,645.7 ns |     40,004.1 ns |  1.00 |    0.01 |          - |          - |          NA |
|        |            |                   |                                      |                 |                 |                 |                 |                 |                 |                 |       |         |            |            |             |
|  **Count** | **Job-IXUFSP** | **\main\corerun.exe** |                                  **zqj** | **NonBacktracking** |     **37,983.6 ns** |       **652.88 ns** |       **610.70 ns** |     **38,011.5 ns** |     **36,730.5 ns** |     **39,293.3 ns** |  **1.00** |    **0.00** |          **-** |          **-** |          **NA** |
|  Count | Job-OTMQBB |   \pr\corerun.exe |                                  zqj | NonBacktracking |     37,807.8 ns |       711.61 ns |       630.83 ns |     37,738.9 ns |     36,592.4 ns |     39,037.1 ns |  0.99 |    0.02 |          - |          - |          NA |

Reply:
Ok, thanks.